catalog/opensearch/catalog-opensearch-source/src/test/java/ddf/catalog/source/opensearch/impl/TestOpenSearchSource.java

```diff
@@ -697,15 +780,38 @@ protected SecureCxfClientFactory createClientFactory(
     }
   }
 
-  protected SecureCxfClientFactory getMockFactory(WebClient client) {
-    SecureCxfClientFactory factory = mock(SecureCxfClientFactory.class);
+  //    private class OverriddenOpenSearchSource extends OpenSearchSource {
```

Comment:
❓ Why is this chunk commented out?

Reply:
oops, sorry about that, leftovers from figuring out the rebasing